core/app/modules/import-export/runners/revert/plugins.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Elementor\Core\App\Modules\ImportExport\Runners\Revert;
+
+use Elementor\Core\Utils\Collection;
+use Elementor\Core\Utils\Plugins_Manager;
+use Elementor\Core\Utils\Str;
+
+class Plugins extends Revert_Runner_Base {
```

Comment:
Let's add test the if I add a new runner it is included in all 3 - import/export/revert

Reply:
I don't think I can